### PR TITLE
Clojars support

### PR DESCRIPTION
**Pull request by erwan**
For Clojure, see: http://clojars.org/

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1088,6 +1088,33 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Clojars version integration
+camp.route(/^\/clojars\/v\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var repo = match[1];  // eg, `prismic`.
+  var format = match[2];
+  var apiUrl = 'https://clojars.org/search?q=' + repo + '&format=json';
```
Comment (espadrine) [inline server.js]: Could you escape `repo` please? Something like `encodeURIComponent(repo)` might be enough.

**Comment by espadrine** [comment]
Thanks a lot for this contribution!

**Comment by dotemacs** [comment]
This suggested change will do a search every time a clojar is looked up, which isn't very efficient.

So I've implemented a change which has been merged, allowing you to look up a URL for given package, that looks like:

http://clojars.org/some-clojar-name/latest-version.json

where **some-clojar-name** is a valid clojar name. 

This feature will be available on the next release. So @erwan do you want to re-implement this PR or shall I do it? Thanks

**Comment by erwan** [comment]
You can do it @dotemacs  it's fine for me!

**Comment by dotemacs** [comment]
OK, I'll try to tackle it tonight/tomorrow! Thank you @erwan

**Pull request merged**